src/utils/llms.py

```diff
@@ -21,16 +21,16 @@
 
 # Google
 GEMINI = ChatGoogleGenerativeAI(
-    model="gemini-2.5-pro",
-    temperature=0.0,
-    max_tokens=None,  # max_tokens=None means no limit
-    include_thoughts=False,
+    model="gemini-3-pro-preview",
+    temperature=1.0,
+    max_tokens=None,
+    include_thoughts=True,
```

Comment:
We are not using thoughts in FE - keeping this False, keeps the API consistent (although, I think we have filter set in FE for thinking tokens)